src/app/routes/getInitialData/utils/getPreprocessorRules/index.js

```diff
@@ -0,0 +1,29 @@
+import {
+  cpsAssetPreprocessorRules,
+  articlesPreprocessorRules,
+  indexPreprocessorRules,
+  radioPagePreprocessorRules,
+} from '../preprocessorRulesConfig';
+
+const getPreprocessorRules = type => {
+  switch (type) {
+    case 'article':
+      return articlesPreprocessorRules;
+    case 'WS-LIVE':
+      return radioPagePreprocessorRules;
+    case 'IDX':
+      return indexPreprocessorRules;
+    case 'FIX':
+      return indexPreprocessorRules;
```

Comment:
It could be argued that `indexPreprocessorRules` could be called `frontPagePreprocessorRules`, but I feel like both are an index rather than a FIX should be treated directly as an IDX.